src/listeners/automod/automodPhish.ts

```diff
@@ -47,6 +47,23 @@ export const automodPhish: ListenerHandler = async (
     let scamSource = "";
 
     for (const link of blockedLinkList) {
+
+      const checkHeptagramAPI = await axios.get<boolean>(
+        `http://heptagrambotproject.com/api/v0/api/scam/link/check?url=${link}`,
+        {
+          // send authentication header
+          headers: {
+            Authorization: "Bearer " + config.heptagramApiToken,
+          },
+        });
```

Comment:
```suggestion
    for (const link of blockedLinkList) {
      const checkHeptagramAPI = await axios.get<boolean>(
        `http://heptagrambotproject.com/api/v0/api/scam/link/check?url=${link}`,
        {
          // send authentication header
          headers: {
            Authorization: "Bearer " + Becca.configs.heptagramApiToken,
          },
        }
      );
```
Linter fix + update the API token location.